From 68874d652f2e82da56cb05ec1e998d2f5d600a69 Mon Sep 17 00:00:00 2001
From: Suman Anna <s-anna@ti.com>
Date: Mon, 24 Apr 2023 09:03:11 -0500
Subject: [PATCH 462/508] samples/rpmsg: Add compatible to support TI IPC
 firmwares

Add a specific compatible "ti.ipc4.ping-pong" to the rpmsg device id
list of the rpmsg client sample driver so that it can be properly
probed with the rpmsg devices published and used by the current TI PDK IPC
sample firmware images. This provides a kernel-level unit-test for the
basic rpmsg communication with the various remote processors on various TI
K3 SoCs like AM65x, J721E, J7200 etc.

Signed-off-by: Suman Anna <s-anna@ti.com>
Signed-off-by: Devarsh Thakkar <devarsht@ti.com>
Signed-off-by: Vignesh Raghavendra <vigneshr@ti.com>
---
 samples/rpmsg/rpmsg_client_sample.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/samples/rpmsg/rpmsg_client_sample.c b/samples/rpmsg/rpmsg_client_sample.c
index ae5081662283..9736aef7f7cc 100644
--- a/samples/rpmsg/rpmsg_client_sample.c
+++ b/samples/rpmsg/rpmsg_client_sample.c
@@ -79,6 +79,7 @@ static void rpmsg_sample_remove(struct rpmsg_device *rpdev)
 
 static struct rpmsg_device_id rpmsg_driver_sample_id_table[] = {
 	{ .name	= "rpmsg-client-sample" },
+	{ .name = "ti.ipc4.ping-pong" },
 	{ },
 };
 MODULE_DEVICE_TABLE(rpmsg, rpmsg_driver_sample_id_table);
-- 
2.41.0

